socorro/middleware/middleware_app.py

```diff
@@ -65,6 +65,7 @@
     (r'/releases/(channels|featured|release)/(.*)', 'releases.Releases'),
     (r'/report/(list)/(.*)', 'report.Report'),
     (r'/server_status/(.*)', 'server_status.ServerStatus'),
+    (r'/signature/first_date', 'signature_first_date.SignatureFirstDate'),
```

Comment:
I wonder why all the other ones have `(.*)` at the end? Sigh! Can't wait to rewrite this.